prod-devel-rcar-virtio.yaml

```diff
@@ -321,46 +325,52 @@ components:
     sources:
       - type: repo
         url: https://github.com/xen-troops/android_kernel_manifest.git
-        rev: common-android12-5.4-xt-master
+        rev: common-android14-6.1-xenvm-trout-main
         manifest: default.xml
         depth: 1
         groups: "%{XT_DOMA_SOURCE_GROUP}"
         dir: "."
     builder:
-      type: android_kernel
-      env:
-        - "TARGET_BOARD_PLATFORM=%{SOC_FAMILY}"
-        - "BUILD_CONFIG=common/build.config.xenvm"
-        - "SKIP_MRPROPER=1"
-        - "EXT_MODULES=%{XT_DOMA_KERNEL_EXTRA_MODULES}"
+      type: bazel
+      tool: "tools/bazel"
+      command: run
+      args:
+        - "--verbose_failures"
+        - "--sandbox_debug"
+      target: "%{ANDROID_KERNEL_BUILD_CONFIG}"
+      target-patterns:
+        - "--dist_dir=%{ANDROID_KERNEL_DEPLOY_DIR}"
       target_images:
-        - "out/android12-5.4/common/arch/arm64/boot/Image"
-        - "out/android12-5.4/staging/vendor/lib/modules/pvrsrvkm.ko"
+        - "%{ANDROID_KERNEL_DEPLOY_DIR}/Image"
+        - "%{ANDROID_KERNEL_DEPLOY_DIR}/initramfs.img"
```

Comment:
do you really use initramfs.img from kernel build ?

Reply:
Hi @arminn, this dependency once was actual. But at the current moment, we do not unpack initramfs.img. Thus, I've removed it.